hadoop-ozone/tools/pom.xml

```diff
@@ -126,6 +126,12 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
+      <version>5.3.27</version>
```

Comment:
Version is defined here:

https://github.com/apache/ozone/blob/f32e98bbe90fec4e72eb5b7050f173c73132919d/hadoop-ozone/pom.xml#L29

Also, _dependency_ check is failing:

https://github.com/tanvipenumudy/ozone/actions/runs/6703292525/job/18214380212

Reply:
Thank you @adoroszlai, I have added the environment variable instead.

Added a newer entry under `LICENSE.txt` and `jar-report.txt` for fixing the failing dependency checks.